server/command_import.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+	"strings"
+
+	cloud "github.com/mattermost/mattermost-cloud/model"
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/pkg/errors"
+)
+
+func (p *Plugin) runImportCommand(args []string, extra *model.CommandArgs) (*model.CommandResponse, bool, error) {
+	if len(args) == 0 {
+		return nil, true, errors.New("must provide an installation DNS")
+	}
+	installationDNS := standardizeName(args[0])
+
+	splitDNS := strings.Split(installationDNS, ".")
+	if len(splitDNS) < 2 {
+		return nil, true, errors.New("failed to parse DNS value")
+	}
+	name := splitDNS[0]
+
+	cloudInstall, err := p.cloudClient.GetInstallationByDNS(installationDNS, nil)
+	if err != nil {
+		return nil, false, errors.Wrap(err, "failed to get installation by DNS")
+	}
+	if cloudInstall == nil {
+		return nil, true, errors.New("no installation for the DNS provided")
+	}
+	if cloudInstall.OwnerID != extra.UserId {
+		cloudInstall.OwnerID = extra.UserId
+	}
+
+	p.cloudClient.UpdateInstallation(cloudInstall.ID, &cloud.PatchInstallationRequest{
```

Comment:
This call is updating the installation in the provisioner and returning an error. You aren't capturing the error here, so we should do that and check to make sure the error is `nil`.

Also, we only need to run this when `cloudInstall.OwnerID != extra.UserId` which is checked on line 31. We should probably move this inside that `if` statement.